src/configs/environments.js

```diff
@@ -15,6 +15,7 @@ const testDatabase = process.env.TEST_DATABASE;
 const host = process.env.HOST;
 const dialect = 'postgres';
 const port = process.env.DB_PORT;
+const { APP_URL_FRONTEND } = process.env;
```

Comment:
can you tell the reason for ``APP_URL_FRONTEND``

Reply:
this URL is what I assigned to be associated with those URLs so that it can be both opened together, but we will have to change it if we make our Heroku back to life